src/index.html

```diff
@@ -11,7 +11,175 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <div>
+          <label>
+            Surname:
+            <input type="text" name="surname">
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            Name:
+            <input type="text" name="name">
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              name="date"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="agreement"
+              required
+            >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div>
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            Password:
+            <input
+              type="password"
+              name="password"
+              minlength="8"
+              maxlength="30"
+              required
+            >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>
+          An interesting fact about you!
+        </legend>
+
+        <div>
+          <label>
+            Do you love cats?
+            <input type="radio" name="yes">
+            Yes
+            <input type="radio" name="yes">
+            No
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            What is your favorite color?
+            <input type="color" name="color">
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            What time do you go to bed?
+            <input type="time" name="time">
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            What are your favorite brand of cars?
+            <select name="car" multiple>
+              <option name="bmw">BMW</option>
+              <option name="audi">Audi</option>
+              <option name="Lada">Lada</option>
```

Comment:
@IoghiStudio how do you think, what data will be sent to a server on form submit?
names followed by it's values - there's no value for any of the options

also be consistent in letter casing
```suggestion
              <option name="bmw" value...>BMW</option>
              <option name="audi" value...>Audi</option>
              <option name="lada" value...>Lada</option>
```

Reply:
Thanks . I wasn't paying attention! Small details for big bugs!